README.md

```diff
@@ -31,52 +31,56 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question.
 
 1. In 1-2 sentences, explain what React's `useRouteMatch` hook is used for.
+   A: Route Match gives you access to the match information if the route matches.
 1. How would you explain form validation to someone who has never programmed before?
+   A: validation is checking to see if everything works
```

Comment:
I may phrase this a little differently, form field validation tells the user if the information they entered is what the system is expecting.